Makefile

```diff
@@ -6,7 +6,7 @@ MDOX_VALIDATE_CONFIG ?= .mdox.validate.yaml
 # if macos, use gsed
 SED ?= $(shell which gsed 2>/dev/null || which sed)
 
-LINT_DIRS = $(shell go list ./... | grep -v "internal/prometheus")
+LINT_DIRS = $(shell go list ./... | grep -v "^parser")
```

Comment:
Looks like we need to match the entire package, so maybe something like `grep -v "$(go list -m)/parser"`?

Reply:
It worked thanks!